cloud/blockstore/tools/testing/chaos-monkey/main.go

```diff
@@ -93,7 +93,7 @@ func run(ctx context.Context, opts *options) error {
 		}
 
 		for _, o := range outputs {
-			fmt.Printf("cleanup: %v\n", o)
+			log.Printf("cleanup: %v\n", o)
```

Comment:
no need for `\n` when you use `log.Printxxx`
change here and in all other places